### PR TITLE
[ZEPPELIN-4160] Fixed Move this note to trash icon always show

### PR DESCRIPTION
**Pull request by liuxunorg**
### What is this PR for?
when the note moved to trash, the note should show 'Remove this note permanently', it does not work correctly after clicked again, more ~Trash folder will be created

### What type of PR is it?
[Bug Fix]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-4160

### How should this be tested?
* [CI pass](https://travis-ci.org/liuxunorg/zeppelin/builds/532764123)

### Questions:
* Does the licenses files need update? NO
* Is there breaking changes for older versions? NO
* Does this needs documentation? NO

### CODE REVIEW

**Comment by liuxunorg** [comment]
@zjffdu , Please help me review code. Thanks!

File: zeppelin-web/src/app/notebook/notebook.controller.js [hunk 1]
```diff
@@ -237,7 +237,8 @@ function NotebookCtrl($scope, $route, $routeParams, $location, $rootScope,
   };
 
   $scope.isTrash = function(note) {
-    return note ? note.name.split('/')[0] === TRASH_FOLDER_ID : false;
+    console.log('note.path = ' + note.path);
```
Comment (felixcheung) [inline zeppelin-web/src/app/notebook/notebook.controller.js]: remove this before merging?

Comment (liuxunorg) [reply zeppelin-web/src/app/notebook/notebook.controller.js]: ok.

**Comment by liuxunorg** [comment]
> can you explain the change from `[0]` to `[1]`?

@felixcheung , This is because of the problem caused by this [ZEPPELIN-2619](https://github.com/apache/zeppelin/commit/085efeb646f3a95b39bf555b5b04551119e3c0b9).

**Comment by felixcheung** [comment]
ok. do you think we should add more tests for ZEPPELIN-2619?

**Comment by liuxunorg** [comment]
@felixcheung , More busy, I suggest to find out the problem and solve it. :-)

**Comment by liuxunorg** [comment]
Will merge if no more comments

**Pull request closed without merging**